__mocks__/content/index.mdx

```diff
@@ -6,3 +6,9 @@ tags: tag1, tag2, tag3
 # Welcome
 
 [link](blog0.mdx)
+
+- [] uncompleted task 1
```

Comment:
Surely we need a pure list item ... which is not a task and check it is not showing up in our tasks list ...

We are in theory extracting tasks not just all list items

i.e. something like ...

```md
- this is a list item
- this is another list item
```

Reply:
Yes, of course, I tested for those before committing.

Actually, this exact line `- [] uncompleted task 1` is a list item that's not being identified as a task because it's `[]` and not `[ ]`, i.e., missing a space between the brackets, thus not a valid task (not valid in Obsidian, Github markdown, or even remark-parse).

Reply:
@mohamedsalem401 ok that wasn't obvious because of the name. 😄